pkg/bootstrap/docker/host/host.go

```diff
@@ -76,155 +72,45 @@ func (h *HostHelper) EnsureVolumeShare() error {
 	return nil
 }
 
-func catHostFile(hostFile string) string {
-	file := path.Join("/rootfs", hostFile)
-	return fmt.Sprintf("cat %s", file)
-}
-
-// CopyFromHost copies a set of files from the Docker host to the local file system
-func (h *HostHelper) CopyFromHost(sourceDir, destDir string) error {
+// DownloadDirFromContainer copies a set of files from the Docker host to the local file system
+func (h *HostHelper) DownloadDirFromContainer(sourceDir, destDir string) error {
```

Comment:
I see. Can we pass the bindmounts in and have a wrapper function that passes the config one in?